public/app/plugins/panel/state-timeline/timeline.ts

```diff
@@ -382,7 +381,7 @@ export function getConfig(opts: TimelineCoreOptions) {
     hovered[i] = o;
   }
 
-  function hoverMulti(cx: number, cy: number) {
+  function highlightMulti(cx: number, cy: number) {
```

Comment:
@leeoniya I've done couple of naming changes here. I want to separate tooltip from highlight and this is the first step. Happy to hear some thoughts!

Reply:
renames look mostly fine to me